### PR TITLE
Staging

### PR DESCRIPTION
**Pull request by gopalakrishnanr-paypaycorp**

### CODE REVIEW

File: jp/content.json [hunk 1]
```diff
@@ -7038,24 +7038,29 @@
...
+                "loadingAnimtnData1_colo2":[
+                  {"desc":["<a href='https://paypay-corp.rickcloud.jp/wiki/display/PayPayDesign/Typography#Typography-textsize' target='_blank'>Body 12</a>","Text Align-Center","","#606060"]}
+                ]
               },
               {
                 "name":"panel4",
                 "loadingAnimtnData1_colo1_title": "Buttons",
-                "loadingAnimtnData1_colo2":[{"desc":["Buttons",""]}]
+                "loadingAnimtnData1_colo2":[
+                  {"desc":["<a href='https://paypay-corp.rickcloud.jp/wiki/display/PayPayDesign/Buttons' target='_blank'>Buttons</a>",""]}
```
Comment (shreyanshp) [inline jp/content.json:7062]: remove wiki

**Review by shreyanshp — CHANGES_REQUESTED** [review]

File: jp/content.json (hunk 1 above)
Comment (gopalakrishnanr-paypaycorp) [reply jp/content.json]: links are removed.

File: jp/content.json [hunk 2]
```diff
@@ -7038,24 +7038,29 @@
                 "name":"panel1",
                 "loadingAnimtnData1_colo1_title": "Animation",
                 "loadingAnimtnData1_colo2":[
-                  {"desc":["","150*15px/dp","Lottie animation"]}]
+                  {"desc":["","150*15px/dp","<a href='https://lottiefiles.com/share/JHROo3' target='_blank'>Lottie animation</a>"]}
```
Comment (shreyanshp) [inline jp/content.json:7041]: Why is this pointing to lottiefiles?

Comment (gopalakrishnanr-paypaycorp) [reply jp/content.json]: @shreyanshp yes we got approval to remove those link and i have updated.(confired by sunder and Yagi san)

Comment (gopalakrishnanr-paypaycorp) [reply jp/content.json]: Kindly approve and merge this PR.

**Review by shreyanshp — APPROVED** [review]

**Pull request merged**